### DISCUSSION TITLE
Hover icon toolbar

### DISCUSSION THREAD

**Discussion by dannypieterss** [tarampampam/random-user-agent#363] [Ideas] [answered]
Hello, 

I was wondering if you could set the current useragent as hovering text so you can see the current while hovering the icon on the toolbar.

Thanks for your cool addon!

Greetings Danny
![hover text](https://user-images.githubusercontent.com/19880453/196868609-94c676b5-ec00-4ae2-b2ab-cb89de70332b.png)

**Comment by tarampampam** [comment, marked as answer]
Moved into issues: https://github.com/tarampampam/random-user-agent/issues/364
